test/ttmlir/Conversion/StableHLOToTTIR/sign_op.mlir

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module @jit_eltwise_sign attributes {} {
+  func.func public @test_sign(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = stablehlo.sign %arg0 : tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
```

Comment:
Can you please improve these checks in tests that you've added? See this message: https://tenstorrent.slack.com/archives/C07A30HRX7C/p1727698795548319

Reply:
Noted. The more detailed checks in tests have been added in this and other mentioned PRs.